tests/FSharp.Editing.Tests/GoToDefinitionTests.fs

```diff
@@ -260,7 +260,7 @@ type Console =
     static member Beep : unit -> unit
 """
 
-[<Test>]
+[<Test; Ignore "Need extra xml files on Travis">]
```

Comment:
@dungpa do we want to do Assert.Inconclusive instead? This would enable to keep the test runing and be either green or ignored.

We could also use Inconclusive only when we are running through Travis so it would still catch regressions on AppVeyor.